### ISSUE TITLE
edge case failures with `dplyr 1.0.0` 

### ISSUE THREAD

**Issue by IndrajeetPatil** [IndrajeetPatil/groupedstats#25]
``` r
library(tidyverse)

df <- tibble::as_tibble(mtcars, rownames = "name") %>%
  dplyr::rename(.data = ., n = wt)

groupedstats::grouped_summary(df, am, n)
#> Using `n` as weighting variable
#> Error: Column 'n' is already present in output
#>  * Use `name = "new_name"` to pick a new name
```

<sup>Created on 2020-03-23 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> - Session info ---------------------------------------------------------------
#>  setting  value                                             
#>  version  R Under development (unstable) (2020-02-28 r77874)
#>  os       Windows 10 x64                                    
#>  system   x86_64, mingw32                                   
#>  ui       RTerm                                             
#>  language (EN)                                              
#>  collate  English_United States.1252                        
#>  ctype    English_United States.1252                        
#>  tz       Europe/Berlin                                     
#>  date     2020-03-23                                        
#> 
#> - Packages -------------------------------------------------------------------
#>  ! package      * version      date       lib
#>    assertthat     0.2.1        2019-03-21 [1]
#>    backports      1.1.5        2019-10-02 [1]
#>    base64enc      0.1-3        2015-07-28 [1]
#>    bayestestR     0.5.2.1      2020-03-16 [1]
#>    boot           1.3-24       2019-12-20 [2]
#>    broom          0.5.3.9000   2020-03-01 [1]
#>    broom.mixed    0.2.4.9000   2020-03-09 [1]
#>    broomExtra     2.5.0        2020-03-17 [1]
#>    callr          3.4.2        2020-02-12 [1]
#>    cellranger     1.1.0        2016-07-27 [1]
#>    cli            2.0.2        2020-02-28 [1]
#>    coda           0.19-3       2019-07-05 [1]
#>    codetools      0.2-16       2018-12-24 [2]
#>    colorspace     1.4-1        2019-03-18 [1]
#>    crayon         1.3.4        2017-09-16 [1]
#>    DBI            1.1.0        2019-12-15 [1]
#>    dbplyr         1.4.2        2019-06-17 [1]
#>    desc           1.2.0        2018-05-01 [1]
#>    devtools       2.2.2        2020-02-17 [1]
#>    digest         0.6.25       2020-02-23 [1]
#>    dplyr        * 0.8.99.9002  2020-03-23 [1]
#>    effectsize     0.3.0        2020-03-22 [1]
#>    ellipsis       0.3.0        2019-09-20 [1]
#>    emmeans        1.4.5        2020-03-04 [1]
#>    estimability   1.3          2018-02-11 [1]
#>    evaluate       0.14         2019-05-28 [1]
#>    fansi          0.4.1        2020-01-08 [1]
#>    forcats      * 0.5.0        2020-03-01 [1]
#>    fs             1.3.2        2020-03-05 [1]
#>    generics       0.0.2        2018-11-29 [1]
#>    ggplot2      * 3.3.0        2020-03-05 [1]
#>    glue           1.3.2        2020-03-12 [1]
#>    groupedstats   0.2.2        2020-03-23 [1]
#>    gtable         0.3.0        2019-03-25 [1]
#>    haven          2.2.0        2019-11-08 [1]
#>    highr          0.8          2019-03-20 [1]
#>    hms            0.5.3        2020-01-08 [1]
#>    htmltools      0.4.0        2019-10-04 [1]
#>    httr           1.4.1        2019-08-05 [1]
#>    insight        0.8.2.1      2020-03-22 [1]
#>    ipmisc         1.2.0.9000   2020-03-05 [1]
#>    jsonlite       1.6.1        2020-02-02 [1]
#>    knitr          1.28         2020-02-06 [1]
#>    lattice        0.20-40      2020-02-19 [2]
#>    lifecycle      0.2.0.9000   2020-03-16 [1]
#>    lme4           1.1-22       2020-03-23 [1]
#>    lubridate      1.7.4        2018-04-11 [1]
#>    magrittr       1.5          2014-11-22 [1]
#>    MASS           7.3-51.5     2019-12-20 [2]
#>    Matrix         1.2-18       2019-11-27 [2]
#>    memoise        1.1.0        2017-04-21 [1]
#>    minqa          1.2.4        2014-10-09 [1]
#>    modelr         0.1.6        2020-02-22 [1]
#>    multcomp       1.4-12       2020-01-10 [1]
#>    munsell        0.5.0        2018-06-12 [1]
#>    mvtnorm        1.1-0        2020-02-24 [1]
#>    nlme           3.1-144      2020-02-06 [2]
#>    nloptr         1.2.2.1      2020-03-11 [1]
#>    parameters     0.6.0        2020-03-12 [1]
#>    performance    0.4.4.1      2020-03-11 [1]
#>    pillar         1.4.3        2019-12-20 [1]
#>    pkgbuild       1.0.6        2019-10-09 [1]
#>    pkgconfig      2.0.3        2019-09-22 [1]
#>    pkgload        1.0.2        2018-10-29 [1]
#>    plyr           1.8.6        2020-03-03 [1]
#>    prettyunits    1.1.1        2020-01-24 [1]
#>    processx       3.4.2        2020-02-09 [1]
#>    ps             1.3.2        2020-02-13 [1]
#>    purrr        * 0.3.3        2019-10-18 [1]
#>    R6             2.4.1        2019-11-12 [1]
#>    Rcpp           1.0.4        2020-03-17 [1]
#>    readr        * 1.3.1        2018-12-21 [1]
#>    readxl         1.3.1        2019-03-13 [1]
#>    remotes        2.1.1        2020-02-15 [1]
#>    repr           1.1.0        2020-01-28 [1]
#>    reprex         0.3.0        2019-05-16 [1]
#>    reshape2       1.4.3        2017-12-11 [1]
#>    rlang          0.4.5.9000   2020-03-23 [1]
#>    rmarkdown      2.1          2020-01-20 [1]
#>    rprojroot      1.3-2        2018-01-03 [1]
#>    rstudioapi     0.11         2020-02-07 [1]
#>    rvest          0.3.5        2019-11-08 [1]
#>    sandwich       2.5-1        2019-04-06 [1]
#>    scales         1.1.0        2019-11-18 [1]
#>    sessioninfo    1.1.1        2018-11-05 [1]
#>    sjlabelled     1.1.3        2020-01-28 [1]
#>    sjmisc         2.8.3        2020-01-10 [1]
#>    sjstats        0.17.9       2020-02-06 [1]
#>    skimr          2.1          2020-02-01 [1]
#>    statmod        1.4.34       2020-02-17 [1]
#>    stringi        1.4.6        2020-02-17 [1]
#>    stringr      * 1.4.0        2019-02-10 [1]
#>    survival       3.1-8        2019-12-03 [2]
#>    testthat       2.3.2        2020-03-02 [1]
#>    TH.data        1.0-10       2019-01-21 [1]
#>    tibble       * 2.99.99.9014 2020-03-23 [1]
#>    tidyr        * 1.0.2        2020-01-24 [1]
#>    tidyselect     1.0.0        2020-01-27 [1]
#>    tidyverse    * 1.3.0        2019-11-21 [1]
#>  D TMB            1.7.16       2020-01-15 [1]
#>    usethis        1.5.1.9000   2020-03-18 [1]
#>    vctrs          0.2.99.9010  2020-03-23 [1]
#>    withr          2.1.2        2018-03-15 [1]
#>    xfun           0.12         2020-01-13 [1]
#>    xml2           1.2.5        2020-03-11 [1]
#>    xtable         1.8-4        2019-04-21 [1]
#>    yaml           2.2.1        2020-02-01 [1]
#>    zeallot        0.1.0        2018-01-28 [1]
#>    zoo            1.8-7        2020-01-10 [1]
#>  source                                
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (easystats/bayestestR@6ee7e37) 
#>  CRAN (R 4.0.0)                        
#>  Github (tidymodels/broom@3c922d5)     
#>  Github (bbolker/broom.mixed@c2587ef)  
#>  local                                 
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (tidyverse/dplyr@35d3ace)      
#>  Github (easystats/effectsize@6f4d5a3) 
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  local                                 
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (easystats/insight@b46a9eb)    
#>  local                                 
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (r-lib/lifecycle@355dcba)      
#>  Github (lme4/lme4@531c361)            
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (easystats/performance@913fee0)
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (r-lib/rlang@a90b04b)          
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (tidyverse/tibble@96af653)     
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  Github (r-lib/usethis@8c32c73)        
#>  Github (r-lib/vctrs@3675fdf)          
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#>  CRAN (R 4.0.0)                        
#> 
#> [1] C:/Users/inp099/Documents/R/win-library/4.0
#> [2] C:/Program Files/R/R-devel/library
#> 
#>  D -- DLL MD5 mismatch, broken installation.
```

</details>

Trace:
```r
16.
stop(fallback) 
15.
signal_abort(cnd) 
14.
.abort(text) 
13.
glubort("Column 'n' is already present in output\n", "* Use `name = \"new_name\"` to pick a new name") 
12.
check_name(x, name) 
11.
dplyr::tally(df_results) 
10.
is.data.frame(y) 
9.
same_src.data.frame(x, y) 
8.
same_src(x, y) 
7.
auto_copy(x, y, copy = copy) 
6.
left_join.data.frame(x = df_results %>% dplyr::group_modify(.f = ~tibble::as_tibble(skimr::skim(purrr::keep(.x = ., 
    .p = ..f))), keep = FALSE) %>% dplyr::ungroup(x = .), y = dplyr::tally(df_results), 
    by = purrr::map_chr(.x = grouping.vars, .f = rlang::as_string)) 
5.
dplyr::left_join(x = df_results %>% dplyr::group_modify(.f = ~tibble::as_tibble(skimr::skim(purrr::keep(.x = ., 
    .p = ..f))), keep = FALSE) %>% dplyr::ungroup(x = .), y = dplyr::tally(df_results), 
    by = purrr::map_chr(.x = grouping.vars, .f = rlang::as_string)) 
4.
eval(lhs, parent, parent) 
3.
eval(lhs, parent, parent) 
2.
dplyr::left_join(x = df_results %>% dplyr::group_modify(.f = ~tibble::as_tibble(skimr::skim(purrr::keep(.x = ., 
    .p = ..f))), keep = FALSE) %>% dplyr::ungroup(x = .), y = dplyr::tally(df_results), 
    by = purrr::map_chr(.x = grouping.vars, .f = rlang::as_string)) %>% 
    dplyr::mutate(.data = ., n = n - n_missing) %>% purrr::set_names(x = .,  ... at grouped_summary.R#128
1.
groupedstats::grouped_summary(df, am, n) 
```

**Comment by IndrajeetPatil** [comment]
closed in 12199140ad0572e0f90c243b1a4aff571c83d196

**Comment by IndrajeetPatil** [comment]
```r
library(tidyverse)

df <- tibble::as_tibble(mtcars, rownames = "name") %>%
  dplyr::rename(.data = ., n = wt)

groupedstats::grouped_summary(df, am, n)
#> Registered S3 method overwritten by 'broom.mixed':
#>   method      from 
#>   tidy.gamlss broom
#> # A tibble: 2 x 16
#>      am skim_type skim_variable missing complete  mean    sd   min   p25 median
#>   <dbl> <chr>     <chr>           <int>    <dbl> <dbl> <dbl> <dbl> <dbl>  <dbl>
#> 1     0 numeric   n                   0        1  3.77 0.777  2.46  3.44   3.52
#> 2     1 numeric   n                   0        1  2.41 0.617  1.51  1.94   2.32
#> # ... with 6 more variables: p75 <dbl>, max <dbl>, n <int>, std.error <dbl>,
#> #   mean.conf.low <dbl>, mean.conf.high <dbl>
```
